handbook/engineering/distribution/goals.md

```diff
@@ -64,17 +64,17 @@ We will start by looking at our Kubernetes deployment and working on an easier u
   - Design customization workflow.
     - Potentially Kustomize would still be used for last-mile changes and non-standard derivations.
 
-### Improve the debugging and troubleshooting process
+### Further standardize the debugging and troubleshooting process
```

Comment:
I am not content with how under-specified this goal is, I would suggest specifying it further (as I have done here) or removing it from this page until we can specify it further.

I also felt the prior description could leave anyone coming across this page to believe that we have no consistent tooling to debug issues, but this does a great disservice to the fact that we have been investing in exactly that (standardized Grafana, Jaeger, alerting, etc. across all deployments) for the past year and have made great strides there :)

I agree there is more to do, though, so I do think this is a good goal - we should just try to specify it further. I have tried to do that here.

Reply:
This was "intentionally" under-specific as it was what we knew at the time, and its a TBD goal in many of its aspects.
This goal as discussed at the time was not about our monitoring tools, it was about a set of tools/scripts/etc we can provide to customers to bundle their logs/metrics/etc and send them to us (configuration, type, size, diff from upstream, etc) and the process to capture what issues are related to, so we can prioritize improving X.

I think this change fundamentally changes the goal and would split this into its own PR.

Reply:
In this case, I will keep both the original goal here in its under-specified state *and* the new goal I have listed here, and we can improve/merge them later if that sounds good

Reply:
nvm, I will just keep the old one - having both is confusing.